src/scss/colors.scss

```diff
@@ -0,0 +1,25 @@
+//Main colors
+$game-background: #eee8ea;
+$accent-color: #591D2D;
+$accent-color-hover: #73263A;
+
+//Text colors
+$text: #4d4d4d;
+$text-light: #666666;
+$tile-text: #ffffff;
+$title: #1a1a1a;
+$title-hover: #591D2D;
+$sub-title: #bda5ab; 
+$sbu-title-light: #cdbbc0;
+
+//Board
+$board-background: #cdbbc0;
+$double-letter: #76A67C;
```

Comment:
Feels like naming should be reversed for letter and work - i.e. `letter-double` and `word-double`, etc